backend/templates/emails/reminder-email.html

```diff
@@ -0,0 +1,84 @@
+        Groups platform from the previous system, G.O. Penn, to Penn Labs' platform, Penn Clubs.</b>
+    All official registry of student groups, including annual student group registration process, will now be handled
+    through Penn Clubs.
+</p>
+
+<p style="font-size: 1.2em">
+    Additionally, the Student Activities Council (SAC) is partnering with Penn Clubs to host a
+    virtual multiday Activities Fair from <b>Sep 1 – 3</b>. More details on the Fair will be shared at a
+    later date by SAC. To participate in the Fair, you must be a student-run club at the
+    University of Pennsylvania approved by the OSA. Registration through the Penn Clubs portal will require PennKey
+    authentication.
+    The deadline to sign up for the SAC Fair is <b>August 24th</b>.
+</p>
+
+<p style="font-size: 1.2em">
+    Once the request for registration is approved, the group will be listed on the
+    front page of Penn Clubs and will be searchable for students on the site.
+</p>
+
+<p style="font-size: 1.2em">
+    <b>IMPORTANT NOTE:</b> Since this information is being distributed through various listservs and methods, please
+    communicate
+    with your entire executive board before signing up for the fair. <b>ONLY ONE</b> officer of the club
+    should be designated to register the group.
+
+</p>
+
+<p style="font-size: 1.2em">
+    <u><b>Re-Registration Process (including SAC Fair sign-up) - Deadline Aug. 24th</b></u>
+</p>
+
+<p style="font-size: 1.2em">
+    Again, it is imperative that your group designate <b>ONLY ONE</b> executive board member to complete this process,
+    preferably a
+    member who is listed on pennclubs.com as an officer for your club.
+</p>
+
+<p style="font-size: 1.2em">
+    To re-register a returning student group, you can access the Penn Clubs renewal site at <a
+        href="https://pennclubs.com/club/pennlabs/renew">https://pennclubs.com/club/pennlabs/renew</a>.
+    This will show you all the clubs of which you are an administrator.
+</p>
+
+<p style="font-size: 1.2em">
+    If you are logged in, and do not see any clubs, email contact@pennclubs.com with your PennKey and club name to get
+    access to re-register your club for the 2020-2021 school year.
+</p>
+
+<p style="font-size: 1.2em">
+    Follow the 6 steps listed to update your club's information. The final page of the renewal process will ask whether
+    or
+    not your club is interested in participating in the Fall 2020 SAC Fair. To sign up, please simply check the box on
+    this
+    page, and your information will be sent to the SAC Fair Coordinator.
+</p>
+
+<p style="font-size: 1.2em">
+    If you have any questions or problems regarding re-registration, please send an email to contact@pennclubs.com.
+</p>
+
+<p style="font-size: 1.2em">
+    If you have questions specifically regarding the SAC Fair, please send an email to sacfair@sacfunded.net.com.
```

Comment:
I'm not sure that `sacfair@sacfunded.net.com` is a valid email, I think it might be just `sacfair@sacfunded.net`.